OsmAnd/src/net/osmand/plus/views/layers/geometry/MultiProfileGeometryWay.java

```diff
@@ -45,9 +45,9 @@ public MultiProfileGeometryWay(MultiProfileGeometryWayContext context) {
 		super(context, new MultiProfileGeometryWayDrawer(context));
 	}
 
-	public void drawSegments(Canvas canvas, RotatedTileBox tileBox) {
+	public void drawSegments(Canvas canvas, RotatedTileBox tileBox, boolean use3DVisualization) {
```

Comment:
Do we need to pass the use3DVisualization flag in all these methods? We can probably try to do similar to GpxGeometryWay.drawDirectionArrows